frontend/dist/clouwaypush.js

```diff
@@ -156,21 +119,38 @@ angular.module('clouway-push', [])
 
       var subscribeForEvent = function (eventName, correlationId) {
         var subscriber = service.openConnection();
-        connectionMethods.bind(subscriber, eventName, correlationId);
+        var params = {
+          subscriber: subscriber,
+          eventName: eventName,
+          correlationId: correlationId
+        };
+
+        $http.put(endpoints.serviceUrl, '', {params: params}).then(function (data) {
```

Comment:
I've taken this from the GWT implementation. If a connection is not opened yet, then it will be opened automatically during the first event subscription.

But `openConnection`'s logic is executed only once - when there is no subscriber set(or generated). Any subsequent calls are noop-s.